src/modules/partners/partner_test.ts

```diff
@@ -0,0 +1,19 @@
+import request from 'supertest'
+import app from '../../server'
+
+describe('test module partner', () => {
+  it('findAll', async () => {
+    const response = await request(app)
+      .get('/v1/partners')
+
+    expect(response.statusCode).toEqual(200)
+  })
+
+  it('findAll with requestQuery', async () => {
+    const response = await request(app)
+      .get('/v1/partners')
+      .query({ name: 'test' })
+
+    expect(response.statusCode).toEqual(200)
```

Comment:
I don't know the convention used in JDS, but I think tests' names should describe what it does, what's the condition, and what it expects. This way, reader only needs to read the test's name and will understand the purpose of the test.

Reply:
Sorry, I'm also still learning for testing, here I give a simple example, friends, you can develop it again :smile: